### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [I Hate Money/I Hate Money](https://hosted.weblate.org/projects/i-hate-money/i-hate-money/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: ihatemoney/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -84,7 +84,7 @@ msgstr ""
 "choisir un autre"
 
 msgid "Which is a real currency: Euro or Petro dollar?"
-msgstr "Euro ou Petrodollar ?"
+msgstr "Quelle la vraie monnaie : Euro ou Petrodollar ?"
```
Comment (greedybro) [inline ihatemoney/translations/fr/LC_MESSAGES/messages.po:87]: The correct french is "Quelle est la vraie monnaie"

Comment (Glandos) [reply ihatemoney/translations/fr/LC_MESSAGES/messages.po:87]: :facepalm: 
Thanks reminding (kindly) that I shouldn't do 2 or 3 things at the same time.

Comment (greedybro) [reply ihatemoney/translations/fr/LC_MESSAGES/messages.po:87]: Sorry if it was rude, didn't mean to be. I was doing several things at the same time as well

Comment (Glandos) [reply ihatemoney/translations/fr/LC_MESSAGES/messages.po:87]: It wasn't!

**Pull request merged**